docs/02-app/01-building-your-application/08-deploying/01-production-checklist.mdx

```diff
@@ -0,0 +1,123 @@
+
+How to use `@next/bundle-analyzer`:
+
+1. **Install**: Run `npm install @next/bundle-analyzer webpack-bundle-analyzer`.
+2. **Configure**: Add the bundle analyzer's settings to your `next.config.js`.
+
+```js
+const withBundleAnalyzer = require('@next/bundle-analyzer')({
+  enabled: process.env.ANALYZE === 'true',
+})
+module.exports = withBundleAnalyzer({})
+```
+
+3. **Run Analysis**: Use `ANALYZE=true npm run build` to start looking at your file sizes.
+4. **Review Findings**: Check the report to see where you can make improvements.
+
+Doing this regularly while you develop and before you launch your site is important to keep your code efficient.
+
+## Logging
+
+Since Next.js runs on both the client and server, there are multiple forms of logging supported:
+
+- `console.log` in the browser
+- `stdout` on the server
+
+If you want a structured logging package, we recommend [Pino](https://www.npmjs.com/package/pino). If you're using Vercel, there are [pre-built logging integrations](https://vercel.com/integrations?utm_source=next-site&utm_medium=docs&utm_campaign=next-website#logging) compatible with Next.js.
+
+## **Error Handling**
+
+Proper error handling is key to keeping your application stable and providing a good user experience. This is especially important with the dynamic routes and server actions in the App Router.
+
+**Use of `error.js` for Error Handling**:
+
+The `error.js` file convention is used for gracefully handling runtime errors in nested routes. It automatically wraps a route segment and its nested children in a React Error Boundary, allowing you to create a tailored error UI and isolate errors to specific segments while keeping the rest of the application functional.
+
+**Handling Server Errors**:
+
+If an error is thrown inside a Server Component, Next.js forwards an `Error` object to the nearest `error.js` file. This object includes a generic message and a digest property in production as a security measure to prevent leaking sensitive details.
+
+## Loading Performance
+
+To improve loading performance, you first need to determine what to measure and how to measure it. [Core Web Vitals](https://vercel.com/blog/core-web-vitals?utm_source=next-site&utm_medium=docs&utm_campaign=next-website) is a good industry standard that is measured using your own web browser. If you are not familiar with the metrics of Core Web Vitals, review this [blog post](https://vercel.com/blog/core-web-vitals?utm_source=next-site&utm_medium=docs&utm_campaign=next-website) and determine which specific metric/s will be your drivers for loading performance. Ideally, you would want to measure the loading performance in the following environments:
```

Comment:
Suggestion for improving clarity on what core web vitals are, and removing the duplicate blog post link. 

```suggestion
To improve loading performance, you first need to determine what to measure and how to measure it.

[Core Web Vitals](https://vercel.com/blog/core-web-vitals?utm_source=next-site&utm_medium=docs&utm_campaign=next-website) are metrics created by Google to measure the end-user experience of web pages. These metrics are important as they influence Google's ranking. 
```